libs/core/dtos/anime-filter-params.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { AnimeTypeDto } from './anime.dto';
+
+/** Filters params for getting anime dto. */
+export interface AnimeFilterParamsDto {
+
+	/** Selected anime types to display. */
```

Comment:
I would try to avoid any relations to UI layer on our DTOs
```suggestion
	/** Anime types. */
```